actions/add-review-url/README.md

```diff
@@ -0,0 +1,31 @@
+# Azure Dev Spaces GitHub Action for adding the review url on pull request
+add-review-url GitHub Action can be used to write a comment on the pull request which is the app review url hence enabling the testing prior to checkin

```

Comment:
Suggested description:
Azure Dev Spaces can provide reviewers a live, sandboxed preview of the pull request's changes before code is merged into the main branch. The `add-review-url` GitHub Action writes the URL of the *preview app* as a comment on the pull request.

Reply:
Updated